### ISSUE TITLE
Make links look like links

### ISSUE THREAD

**Issue by jeremyboggs** [scholarslab/scholarslab.org#933]
Several areas of the site include links that appear the same as regular text, and lack styles for `:hover`, `:focus`, and other interactive pseudo-classes. An egregious example is the titles of posts on the Blog page.

**Comment by sophiastr** [comment]
i can deal with this issue.
I am interested in contributing to the project you are developing.
Do you still need help?

**Comment by amandavisconti** [comment]
This appears to be fixed with the latest dev design work branch
